[Ready to review] Edit section on open source hardware (#891)
alan-turing-institute/the-turing-way: book/content/open_research/03/openhardware.md

You are not required to post them all, but the more you share the more the community benefits.
There is a lot of crossover here with files to include in open source software projects.

#### Overview and introduction
Your open source hardware project should include a general description of the hardware's identity and purpose, written as much as possible for a general audience.
That is, explain what the project is and what it is for before you get into the technical details.

#### A licence

> An appropriate license on the Open Hardware project and its content grants legal permission to anyone to re-use, modify and distribute the different components of a project according to the terms stated (for example, they must acknowledge your contribution).  

Reviewer: ```suggestion
An appropriate license on the Open Hardware project and its content grant legal permission to anyone to re-use, modify and distribute the different components of a project according to the terms stated (for example, they must acknowledge your contribution).  
```

#### Original design files
These are the original source files that you would use to make modifications to the hardware's design.
The act of sharing these files is the core practice of open source hardware.
- Ideally, your open source hardware project would be designed using a free and open source software application, to maximize the ability of others to view and edit it.

> For better or worse however, hardware design files are often created in proprietary programs and stored in proprietary formats.

Reviewer: ```suggestion
For better or worse, hardware design files are often created in proprietary programs and stored in proprietary formats.
```

It is still essential to share these original design files; they constitute the original "source code" for the hardware.
They are the very files that someone will need in order to contribute changes to a given design.
- Try to make your design files easy for someone else to understand. In particular, organize them in a logical way, comment complex aspects, and note any unusual manufacturing procedures.
- Examples of Original Design Files include 2D drawings and computer-aided design (CAD) files.

#### Auxiliary design files
Beyond the original design files, it is often helpful to share your design in additional, more accessible formats.
For example, best practice open sourcing a CAD design is to share the design not just in its native file format, but also in a range of interchange and export formats that can be opened or imported by other CAD programs.

> - It is also helpful to provide ready-to-view outputs that can easily be viewed by end users who wish to understand (but not necessarily modify) the design.

Reviewer: ```suggestion
- It is also helpful to provide ready-to-view outputs that can easily be viewed by end-users who wish to understand (but not necessarily modify) the design.
```

This will allow others to use it with their hardware or modify it along with their modifications to your hardware.
Document the process required to build your software, including links to any dependencies (for example, third-party libraries or tools). In addition, it is helpful to provide an overview of the state of the software (for example, "stable" or "beta" or "barely-working hack").

#### Photos
Photos help people understand what your project is and how to put it together.
It is good to publish photographs from multiple viewpoints and at various stages of assembly. If you do not have photos, posting 3D renderings of your design is a good alternative. Either way, it is good to provide captions or text that explain what is shown in each image and why it is useful.

#### Instructions and other explanations
In addition to the design files themselves, there are a variety of explanations that are invaluable in helping others to make or modify your hardware:
- Making the hardware: To help others make and modify your hardware design, you should provide instructions for going from your design files to the working physical hardware.

> As part of the instructions, it is helpful to link to datasheets for the components / parts of your hardware and to list the tools required to assemble it.

Reviewer: ```suggestion
As part of the instructions, it is helpful to link to datasheets for the components/parts of your hardware and to list the tools required to assemble it.
```

You are not required to post them all, but the more you share the more the community benefits.
There is a lot of crossover here with files to include in open source software projects.

#### Overview and introduction
Your open source hardware project should include a general description of the hardware's identity and purpose, written as much as possible for a general audience.
That is, explain what the project is and what it is for before you get into the technical details.

> A good photo or rendering can help a lot here.

Reviewer: ```suggestion

```

Reviewer: We can add this requirement in an issue rather than the content

Author: I think the original author (I think this was @mjg211 who can verify?) was suggesting that a photo can be helpful as an accompaniment to the hardware, i.e. in the documentation, rather than a photo is helpful in the Turing Way book?

#### A licence

> This grants legal permission to anyone to re-use, modify, and distribute your designs and hardware according to the terms stated (for example, they must acknowledge your contribution).  

Reviewer: ```suggestion
An appropriate license on the Open Hardware project and its content grants legal permission to anyone to re-use, modify and distribute the different components of a project according to the terms stated (for example, they must acknowledge your contribution).  
```
